index.js

```diff
@@ -436,19 +437,31 @@ protractorUtil.registerJasmineReporter = function(context) {
       }
     },
     specStarted: function() {
-      protractorUtil.test = {
-        start: moment(),
-        specScreenshots: [],
-        specLogs: [],
-        specHtmls: [],
-        failedExpectations: [],
-        passedExpectations: []
-      };
-      protractorUtil.testResults.push(protractorUtil.test);
+        protractorUtil.test = {
+          start: moment(),
+          specScreenshots: [],
+          specLogs: [],
+          specHtmls: [],
+          failedExpectations: [],
+          passedExpectations: [],
+          prefix: ''
+        };
+        global.browser.getProcessedConfig().then(function(config) {
```

Comment:
I think it missing here a return statement

Reply:
`specStarted` doesn't need a return value. Am I right? Therefore, I don't think we need a return statement.